src/Jackett.Common/Indexers/FileListApi.cs

```diff
@@ -0,0 +1,216 @@
+            AddCategoryMapping(23, TorznabCatType.TVSD, "Seriale SD");
+            AddCategoryMapping(13, TorznabCatType.TVSport, "Sport");
+            AddCategoryMapping(12, TorznabCatType.AudioVideo, "Videoclip");
+            AddCategoryMapping(7, TorznabCatType.XXX, "XXX");
+        }
+
+        public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken configJson)
+        {
+            LoadValuesFromJson(configJson);
+
+            var pingResponse = await RequestStringWithCookiesAndRetry(ApiUrl + "?action=latest-torrents", null, null, GetAuthHeader());
+
+            JArray json = null;
+            try
+            {
+                json = JArray.Parse(pingResponse.Content);
+            }
+            catch (Exception ex)
+            {
+                throw new ExceptionWithConfigData(ex.Message, configData);
+            }
+
+            IsConfigured = true;
+
+            SaveConfig();
+
+            return IndexerConfigurationStatus.Completed;
+        }
+
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            var releases = new List<ReleaseInfo>();
+            var response = await CallProviderAsync(query);
+            releases.AddRange(TransformProviderRequest(query, response));
+
+            return releases;
+        }
+
+        private async Task<string> CallProviderAsync(TorznabQuery query)
+        {
+            var searchUrl = ApiUrl;
+            var searchString = query.GetQueryString();
+            var imdbID = query.ImdbID;
+            var cat = MapTorznabCapsToTrackers(query).FirstIfSingleOrDefault("");
+
+            var queryCollection = new NameValueCollection();
+            
+            if (imdbID != null)
+            {
+                queryCollection.Add("type", "imdb");
+                queryCollection.Add("query", imdbID);
+                queryCollection.Add("action", "search-torrents");
+            }
+            else if (!string.IsNullOrWhiteSpace(searchString))
+            {
+                queryCollection.Add("type", "name");
+                queryCollection.Add("query", searchString);
+                queryCollection.Add("action", "search-torrents");
+            } 
+            else 
+            {
+                queryCollection.Add("action", "latest-torrents");
+            }
+
+            queryCollection.Add("category", cat);
+
+
+            searchUrl += "?" + queryCollection.GetQueryString();
+
+            var response = await RequestStringWithCookiesAndRetry(searchUrl, null, null, GetAuthHeader());
+
+            return response.Content;
+        }
+
+        private List<ReleaseInfo> TransformProviderRequest(TorznabQuery query, string results)
+        {
+            var releases = new List<ReleaseInfo>();
+            JArray json = null;
+
+            try
+            {
+                json = JArray.Parse(results);
+                foreach (JObject r in json)
```

Comment:
we prefer to use var instead:
````
                var json = JArray.Parse(results);
                foreach (var r in json)
````

Reply:
general naming convention we're moving towards in the C# trackers is that `r` should be `row`